### ISSUE TITLE
Jest mocks not found on version 1.10.0

### ISSUE THREAD

**Issue by donatoaguirre24** [software-mansion/react-native-gesture-handler#1335]
## Description

Jest does not find `react-native-gesture-handler` mocks on the last version (1.10.0). Probably related to #1199  

### Screenshots

<img width="1507" alt="image" src="https://user-images.githubusercontent.com/20030075/107264721-180efd80-6a22-11eb-9609-465c31a8a802.png">

## Steps To Reproduce

1. Update `react-native-gesture-handler` to version `1.10.0` 
2. Run `yarn test`

### Expected behavior

Run test cases normally as it always does.

### Actual behavior

Fails with error: `Module <rootDir>/node_modules/react-native-gesture-handler/jestSetup.js in the setupFiles option was not found.`

## Package versions

- React: 16.13.1
- React Native: 0.63.4
- React Native Gesture Handler: 1.10.0

**Comment by jakub-gonet** [comment]
Thanks for reporting. Could you write how do you use RNGH mocks? Or you rely on jest automatically mocking RNGH?

**Comment by donatoaguirre24** [comment]
> Thanks for reporting. Could you write how do you use RNGH mocks? Or you rely on jest automatically mocking RNGH?

I have the following in my `jest.config.js` as per the [docs](https://docs.swmansion.com/react-native-gesture-handler/docs/#testing). And do not use the mocks RNGH directly so I guess I rely on jest auto-mocking.

```js
module.exports = {
  preset: 'react-native',
  setupFiles: [
    '<rootDir>/node_modules/react-native-gesture-handler/jestSetup.js',
    '<rootDir>/test/testSetup.js',
  ],
  transformIgnorePatterns: [
    'node_modules/(?!(jest-)?react-native|@react-native-community|@react-navigation)',
  ],
  modulePathIgnorePatterns: ['<rootDir>/e2e/'],
  transform: {
    '^.+\\.js$': 'babel-jest',
  },
};
```

**Comment by joaocarmo** [comment]
I'm having the same issue. I could verify that the file `jestSetup.js` was deleted in https://github.com/software-mansion/react-native-gesture-handler/pull/1199, even though it's listed in the `files` of `package.json`.

**Comment by mym0404** [comment]
With @joaocarmo 's comment,

We can create patch file using `patch-package` like the following with 
named `react-native-gesture-handler+1.10.0.patch`

```
diff --git a/node_modules/react-native-gesture-handler/jestSetup.js b/node_modules/react-native-gesture-handler/jestSetup.js
new file mode 100644
index 0000000..ed990ab
--- /dev/null
+++ b/node_modules/react-native-gesture-handler/jestSetup.js
@@ -0,0 +1 @@
+jest.mock('./dist/src/RNGestureHandlerModule');
\ No newline at end of file

```

**Comment by jakub-gonet** [comment]
@mym0404, does it work when you use module from src/, like in this patch:
```patch
iff --git a/node_modules/react-native-gesture-handler/jestSetup.js b/node_modules/react-native-gesture-handler/jestSetup.js
new file mode 100644
index 0000000..ed990ab
--- /dev/null
+++ b/node_modules/react-native-gesture-handler/jestSetup.js
@@ -0,0 +1 @@
+jest.mock('./src/RNGestureHandlerModule');
\ No newline at end of file
```

**Comment by mym0404** [comment]
@jakub-gonet Yes, it works too with fixing `iff` -> `diff` typo in first line.

**Comment by jakub-gonet** [comment]
Cool, the fix is on master now, after we fix problems with touchables we'll release a patch version shortly.

**Comment by harika-sabbella** [comment]
@jakub-gonet Still not able to use the mock even on latest version 1.10.1 -- I see the following error 
```
TypeError: Cannot read property 'Direction' of undefined

      at Object.<anonymous> (node_modules/react-native-gesture-handler/dist/src/Directions.js:2:33)
      at Object.<anonymous> (node_modules/react-native-gesture-handler/dist/index.js:1:3993)
```

**Comment by artola** [comment]
Having same problem that reported by @jakub-gonet My workaround is using a mock in `jest.setup.js` or similar referenced from the `jest.config.js` as `setupFiles`.

```
jest.mock('react-native-gesture-handler/dist/src/RNGestureHandlerModule');
```

**Comment by lukewlms** [comment]
Also seeing "Cannot read property 'Direction' of undefined" in 1.10.1.  1.9.0 works fine - reverted for the time being.

**Comment by jakub-gonet** [comment]
Fixed in 1.10.2.
